spring-core/src/main/java/org/springframework/cglib/proxy/Enhancer.java

```diff
@@ -1254,11 +1254,11 @@ public ClassLoader getClassLoader() {
 			}
 
 			public int getOriginalModifiers(MethodInfo method) {
-				return ((Integer) originalModifiers.get(method)).intValue();
+				return (Integer) originalModifiers.get(method);
```

Comment:
Please refrain from modifying classes under `org.springframework.asm`, `org.springframework.cglib`, and `org.springframework.objenesis`. Those include repackaged forks of the third-party libraries ASM, CGLIB, and Objenesis. Any refactoring to those classes should take place upstream in the originating repository. The Spring Framework will then pick up the changes when syncing with official updates of the forked third-party libraries.

```suggestion
				return ((Integer) originalModifiers.get(method)).intValue();
```